src/types/public.ts

```diff
@@ -55,3 +55,14 @@ export interface TimeoutsOptions {
    */
   retries?: number;
 }
+
+export interface TaskEnablingOptions {
+  /**
+   * Enable a recurring task while skipping all the past occurrences
+   *
+   * @defaultValue false
+   *
+   * @public
+   */
+  skipPastOccurrences?: boolean;
```

Comment:
what would you think about naming this `recomputeSchedule` or something like it instead? if memory serves, the default behavior isn't to run every occurrence that would have executed, so `skipPastOccurrences` may imply that if it's false that it might queue up multiple runs otherwise

not in love with the name `recomputeSchedule` either, so open to other suggestions

Reply:
Yeah that sounds like a better name. skipPastOccurrences will probably confuse people on the negative case. Ill make that change along with other changes